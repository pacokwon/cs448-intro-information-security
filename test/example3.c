int main()
{
    int x = 0;
    int y = 30;
    int z = divide(x, y);
}

int divide(int a, int b){
    int c = 0;
    if (a >=0){
        c = 30/a;
    }
    else{
        c = 40/a;
    }
    return c;
}