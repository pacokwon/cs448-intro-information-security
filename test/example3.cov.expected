5,13
10,11
10,9
